src/intTest/java/uk/nhs/digital/nhsconnect/nhais/jms/MeshServiceBaseTest.java

```diff
@@ -134,4 +126,17 @@ private MeshMessage deserializeMeshMessage(String meshMessage) {
 
         return dataToReturn.get();
     }
+
+    protected void clearMeshQueue() {
```

Comment:
fix for non-deterministic tests.
I think It's safer here than in MeshClient class.